demos/bluetooth/mesh/thingy52/thingy52_cli/README.rst

```diff
@@ -0,0 +1,214 @@
+.. _bluetooth_mesh_thingy52_cli:
+
+Bluetooth: Mesh Thingy:52 Client
+################################
+
+The Thingy:52 client demo is meant to work together with the :ref:`bluetooth_mesh_thingy52_srv` demo. It is used to inject
+RGB messages into a mesh network consisting of Thingy:52 Server nodes. The client supports message injection with a range
+of different colors, variable message delay, and configurable speaker output.
+
+
+Overview
+********
+
+The Thingy:52 client supports two ways of injecting messages into the mesh network:
+
+* Manual - The user manually selects the color and delay of the message.
+* Auto - When activated by the user the device will inject a message with a random color and delay
+  every two seconds.
+
+The delay contained in a message can be in the range from ``200 ms`` to ``2000 ms``. Both ways of
+message injection can be configured to support speaker output.
+
+Provisioning
+============
+
+Provisioning is handled by the :ref:`bt_mesh_dk_prov`.
+
+Models
+======
+
+The following table shows the Thingy:52 client composition data for this demo:
+
+.. table::
+   :align: center
+
+   +------------------+
+   |   Element 1      |
+   +==================+
+   |  Config Server   |
+   +------------------+
+   |  Health Server   |
+   +------------------+
+   | Thingy:52 Client |
```

Comment:
Is Thingy:52 Client really correct here? I thought the name of the model is Thingy:52..?